docs/source/pillars/computing_technology.md

```diff
@@ -176,79 +232,57 @@ _The ability of the TRE operator to administer and secure network infrastructure
   - Mandatory
 ```
 
-## Infrastructure lifecycle management
-
-_The ability of the TRE operator to manage necessary physical or virtual infrastructure._
+## Capacity management
 
-### Deployment management
+### Capacity planning
 
-_The ability of the TRE operator to instantiate, deploy, change or remove deployed infrastructure._
+The process of forecasting and determining the resources required to meet the demands of an application or system, ensuring that adequate resources are available when needed.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-deployment-management
+:name: tab-infrastructure-capacity
 
 * - Statement
   - Guidance
   - Importance
-* - You must have a documented procedure for deploying infrastructure.
-  - This might, for instance, be a handbook that is followed or a set of automated scripts.
+* - You must ensure that all projects understand what resources are available and what the associated costs will be before the project starts.
+  - For on-premises systems this might be related to the available hardware, for cloud-based systems there might be limits on how many instances of a particular resource (_e.g._ GPUs) can be used. Projects should use this information to understand whether the available resources will be sufficient for their requirements.
   - Mandatory
-* - You should, where possible, automate any repeatable aspects of your deployment.
-  - This might involve using infrastructure-as-code tools or simply a series of scripts.
+* - You should ensure that the anticipated needs of projects can be satisfied using available resources.
+  - Note that this does not require you to accept requests for additional resources, but rather that promises made about resource availability before a project starts should be honoured wherever possible.
   - Recommended
-* - You must have a documented procedure for making changes to deployed infrastructure.
-  - This refers both to changes that might be expected in the course of normal operation and emergency changes that might be needed.
-    Your change management process may form part of a wider accreditation such as ISO 27001.
-  - Mandatory
-* - You must test changes before they are used in production.
-  - This might involve a separate development environment or another system for testing.
-  - Mandatory
-* - You could test changes in a development environment that mirrors your production system.
-  - Consider the costs and practicality of whether this will work for your situation.
-  - Optional
-* - You must have a documented procedure for removing infrastructure when it is no longer needed
-  - Removing unused infrastructure not only reduces costs and management burden but also reduces the attack surface of a TRE and reduces the risk of unaddressed vulnerabilities.
+* - You must have a procedure for increasing/decreasing available resources.
```

Comment:
```suggestion
* - TRES with the ability to increase/decrease available resources must have a documented procedure in place for how to do this.
```

Reply:
A bit of a different tack, but after re-reading the guidance, I think the following might be what we're trying to say here:

```suggestion
* - You must have a procedure for allocating available resources among projects.
```